commit c0394dab3a228d3de9d5ecedb3e57c534886716d
Author: haolong.zhang <haolong.zhang@ck-telecom.com>
Date:   Mon Sep 15 17:01:05 2014 +0800

    [Feature] [WIFI/BT/FACTORY]: Suit CKT SPEC --> WAIT USER HANDLE result of WIFI/BT test
    
        [id]:N/A
        [TYPE]
           FEATURE
        [Products]
           hicam_baseline
        [Requirement]
           N/A
        [Solution Description]
           N/A
        [Other info]
     要提交的变更：
    	修改:         mediatek/factory/src/test/ftm_bt.c
    	修改:         mediatek/factory/src/test/ftm_wifi.c
    
    Change-Id: I3117dadef81e6de9bf2b75809c889cd5486164b2

diff --git a/mediatek/factory/src/test/ftm_bt.c b/mediatek/factory/src/test/ftm_bt.c
index d291e95..b33d413 100644
--- a/mediatek/factory/src/test/ftm_bt.c
+++ b/mediatek/factory/src/test/ftm_bt.c
@@ -52,8 +52,8 @@ enum {
 };
 
 static item_t bf_items[] = {
-//  item(ITEM_PASS,   uistr_pass),
-//  item(ITEM_FAIL,   uistr_fail),
+  item(ITEM_PASS,   uistr_pass),
+  item(ITEM_FAIL,   uistr_fail),
   item(-1, NULL),
 };
 
@@ -134,7 +134,7 @@ exit:
     return NULL;
 }
 
-#if 0
+#if 1
 static int bt_key_handler(int key, void *priv)
 {
     int handled = 0, exit = 0;
@@ -204,7 +204,7 @@ int bt_entry(struct ftm_param *param, void *priv)
     bf->result = false;
     pthread_create(&bf->update_thd, NULL, bt_update_thread, priv);
     
-/*    do {
+   do {
         chosen = iv->run(iv, &exit);
         switch (chosen) {
         case ITEM_PASS:
@@ -222,16 +222,17 @@ int bt_entry(struct ftm_param *param, void *priv)
             bf->exit_thd = true;
             break;
         }
-    } while (1);*/
+    } while (1);
     
     pthread_join(bf->update_thd, NULL);
-    
+ /*   
     if (test_result > 0) {
         bf->mod->test_result = FTM_TEST_PASS;
     }
     else {
         bf->mod->test_result = FTM_TEST_FAIL;
     }
+ */
     return 0;
 }
 
diff --git a/mediatek/factory/src/test/ftm_wifi.c b/mediatek/factory/src/test/ftm_wifi.c
index 2d6c0cd..3a2a432 100644
--- a/mediatek/factory/src/test/ftm_wifi.c
+++ b/mediatek/factory/src/test/ftm_wifi.c
@@ -51,9 +51,9 @@ enum {
 };
 
 static item_t wifi_items[] = {
-/*    item(ITEM_PASS,   uistr_info_wifi_test_pass),
+    item(ITEM_PASS,   uistr_info_wifi_test_pass),
     item(ITEM_FAIL,   uistr_info_wifi_test_fail),
-    item(ITEM_RENEW,  uistr_info_wifi_renew),*/
+    item(ITEM_RENEW,  uistr_info_wifi_renew),
     item(-1, NULL),
 };
 
@@ -225,7 +225,7 @@ int wifi_entry(struct ftm_param *param, void *priv)
     tv->run(tv);
 */
 
-#if 0
+#if 1
     /* initialize thread condition */
     wififm->exit_thd = false;
     wififm->result = false;
